build/templates/errors.rst.mako

```diff
@@ -80,6 +82,17 @@ ${helper.get_rst_header_snippet('SelfTestError', '-')}
         An error due to a failed self-test
 
 
+% endif
+% if grpc_supported:
+${helper.get_rst_header_snippet('RpcError', '-')}
```

Comment:
Why did you choose `RpcError` without the g prefix?
I don't think it matters much, RPC being more generic. But curious. We seem to use the name of the specific technology everywhere explicitly.

Reply:
Well first, I'm just documenting the code that was already submitted. But I understand that seeing it before didn't raise the trigger in your mind. We chose that name because gRPC itself [uses that exception name for Python](https://grpc.github.io/grpc/python/grpc.html#grpc-exceptions). It's also `RpcException` in C# just for reference. I think it's the appropriate choice here, although I do agree with your general guidance.